src/breakout.cpp

```diff
@@ -1,6 +1,6 @@
 #include <iostream>
 
-#include "SDL2/SDL.h"  // NOLINT
+#include "SFML\Graphics.hpp"
```

Comment:
Please use / instead of \ as the forward slash is cross platform but back slash is not. (Windows doesn't care, but Linux will only work with forward slashes)